docs/layouts/shortcodes/generated/table_config_configuration.html

```diff
@@ -62,6 +62,12 @@
             <td>String</td>
             <td>The local time zone defines current session time zone id. It is used when converting to/from &lt;code&gt;TIMESTAMP WITH LOCAL TIME ZONE&lt;/code&gt;. Internally, timestamps with local time zone are always represented in the UTC time zone. However, when converting to data types that don't include a time zone (e.g. TIMESTAMP, TIME, or simply STRING), the session time zone is used during conversion. The input of option is either a full name such as "America/Los_Angeles", or a custom timezone id such as "GMT-08:00".</td>
         </tr>
+        <tr>
+            <td><h5>table.optimizer.query-hints.enabled</h5><br> <span class="label label-primary">Batch</span> <span class="label label-primary">Streaming</span></td>
```

Comment:
Can we please follow what you agreed in the mailing list?

I believe you settled on `table.optimizer.ignore-query-hints`, didn't you? Or has this discussion been inconclusive?

Reply:
Thanks. Actually, let me verify one more time in the discussion since this naming format is also consistent with other existing configs.

Reply:
Updated.

Reply:
cc @wuchong Hey Jark, I am reaching out to you since you suggested the `table.optimizer.ignore-query-hints`. I think Bonnie has a good point here that `table.optimizer.query-hints.enabled` would be more consistent with other configs:
```
table.dynamic-table-options.enabled
table.optimizer.distinct-agg.split.enabled
table.optimizer.dynamic-filtering.enabled
...
```

Moreover, I think `ignore-query-hints` would be a bit misleading. From my point of view it would suggest query hints are simply ignored, but from what I've gathered the agreed behaviour is to throw an exception which is not necessarily ignoring hints. 

Would you be ok with naming the config `table.optimizer.query-hints.enabled`?

Reply:
@bvarghese1 Let's tentatively revert to `table.optimizer.query-hints.enabled`, as I agree with you it is more consistent with other options. After that I'll merge the PR. Thanks for working on that!

Reply:
IMHO this naming leads to misunderstanding of behavior
from SQL point of view content of comments should not impact the output and currently it is (depending on the property value it either returns a result or throws an exception...) Or if we say we set the flag value to false (turn off hint support) then depending on content of comment block in query it either throws exception or returns a valid result...
It looks like it is breaking change and probably violation of SQL Standard...
At the same time jin ML there was mentioned Oracle behavior https://docs.oracle.com/en/database/oracle/oracle-database/23/refrn/OPTIMIZER_IGNORE_HINTS.html#GUID-D62CA6D8-D0D8-4A20-93EA-EEB4B3144347
which returns valid result in both cases. The difference is that whether it takes into account hints or not while building execution plan. 
I'm curious why couldn't we go same way?

Reply:
> from SQL point of view content of comments should not impact the output and currently it is (depending on the property value it either returns a result or throws an exception...) Or if we say we set the flag value to false (turn off hint support) then depending on content of comment block in query it either throws exception or returns a valid result...

I disagree with that take on throwing exception. I think it's exaggerated. Hints are already there and they do impact the output. You can draw comparison with e.g. providing an invalid parameter to e.g. https://nightlies.apache.org/flink/flink-docs-release-1.17/docs/dev/table/sql/queries/hints/#lookup-hint-options If you provide a valid parameter it will produce output, if an invalid one it will throw an exception.

I do agree we should have a community agreement if we want to ignore hints or throw an exception. Another option I see is we can have an enum parameter (ENABLE, IGNORE, THROW).

Reply:
thanks for highlighting this, after checking I confirm that they are impacting output...

about enum probably we could continue in ML